pkg/csi/plugins/nodeserver.go

```diff
@@ -204,6 +204,11 @@ func (ns *nodeServer) NodeUnstageVolume(ctx context.Context, req *csi.NodeUnstag
 	// no volume context attribute.
 	namespace, name, err := ns.getRuntimeNamespacedName(nil, req.GetVolumeId())
 	if err != nil {
+		if utils.IgnoreNotFound(err) == nil {
```

Comment:
In step 2,  Check fuse clean policy. If clean policy is set to OnRuntimeDeleted, there is no need to clean fuse eagerly.  I think we also need think about tolerate the issue that runtime info is missed.